### ISSUE TITLE
Adding Russian language support for interface localization.

### ISSUE THREAD

**Issue by ArtoymKryazhev** [saving-satoshi/saving-satoshi#1194]
I would like to suggest adding support for the Russian language in the game's interface. This will help expand the audience and make the project accessible to Russian-speaking users. Including Russian language localization will improve content accessibility and attract new users.

**Comment by benalleng** [comment]
Absolutely, If you want to begin adding some support for Russian feel free to take a look at the translation README here https://github.com/saving-satoshi/saving-satoshi/blob/dev/i18n/README.md

**Comment by ArtoymKryazhev** [comment]
I figured everything out and have already started translating. Since I am a student I have little time, so I will translate 1-2 chapters once a week. Translated ///ABOUT PAGE and started translating ///CHAPTERS TITLE PAGE
![Screenshot_1](https://github.com/user-attachments/assets/15d7c4f7-e3ae-4437-b2d4-87f0a821cc07)
